app/lib/submission_builder/ty2021/documents/schedule_eic.rb

```diff
@@ -23,7 +23,7 @@ def document
                 xml.ChildIsAStudentUnder24Ind dependent.full_time_student_yes? && dependent.age_during_tax_year < 24
                 xml.ChildPermanentlyDisabledInd dependent.permanently_totally_disabled_yes?
                 xml.ChildRelationshipCd dependent.irs_relationship_enum
-                xml.MonthsChildLivedWithYouCnt '07' # this is a placeholder value until we collect the actual number of months in intake
+                xml.MonthsChildLivedWithYouCnt dependent.months_in_home.to_s.rjust(2, '0')
```

Comment:
nit but we could move this out to a formatting method for IRS month expectations